### PR TITLE
Add Kikai

### PR DESCRIPTION
**Pull request by zehmaluco**
# New Port for { Kikai}

## Game Information
- **Title**: { Kikai }
- **URL**: { https://farawaytimes.itch.io/kikai }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/kikai/Kikai.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+export PORT_32BIT="Y"
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/kikai"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
```
Comment (Cebion) [inline ports/kikai/Kikai.sh:26]: LD_LIBRRAY_PATH export is not correct.

File: ports/kikai/Kikai.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+export GMLOADER_DEPTH_DISABLE=1
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+export GMLOADER_PLATFORM="os_linux"
+
+# We log the execution of the script into log.txt
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
```
Comment (Cebion) [inline ports/kikai/Kikai.sh:32]: Old logging method

File: ports/kikai/Kikai.sh [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+cd $GAMEDIR
+
+if [ -f "${controlfolder}/libgl_${CFWNAME}.txt" ]; then
```
Comment (Cebion) [inline ports/kikai/Kikai.sh:36]: gmloader does not sue libgl

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/kikai/Kikai.sh (hunk 1 above)
Comment (zehmaluco) [reply ports/kikai/Kikai.sh:26]: What is the correct command?

Comment (cdeletre) [reply ports/kikai/Kikai.sh:26]: > What is the correct command?
It should be something like this

`export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs.${DEVICE_ARCH}:$LD_LIBRARY_PATH"`

With `libs` renamed to `libs.armhf` if we stick to the multiarch logic but I'll let @Cebion confirm that as it's a 32bits only port.

File: ports/kikai/Kikai.sh [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+export PORT_32BIT="Y"
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/kikai"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
```
Comment (Cebion) [inline ports/kikai/Kikai.sh:26]:
```suggestion
export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$LD_LIBRARY_PATH"
```

File: ports/kikai/Kikai.sh (hunk 1 above)
Comment (Cebion) [reply ports/kikai/Kikai.sh:26]: > > What is the correct command?
> 
> It should be something like this
> 
> `export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs.${DEVICE_ARCH}:$LD_LIBRARY_PATH"`
> 
> With `libs` renamed to `libs.armhf` if we stick to the multiarch logic but I'll let @Cebion confirm that as it's a 32bits only port.

Correct, thing is that its only 32 bit and nothing else we didn'Tbother with armhf.
So correct would be just ```export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$LD_LIBRARY_PATH"```

Comment (Cebion) [reply ports/kikai/Kikai.sh:26]: or even just export LD_LIBRARY_PATH="$GAMEDIR/libs:$LD_LIBRARY_PATH"

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by Cebion — DISMISSED** [review]

**Comment by zehmaluco** [comment]
@Cebion @cdeletre ths game is now RTR. I've uploded the files, license and updated the port.json. Please check if did everything right. 😄

**Review by kloptops — APPROVED** [review]
Great work. 👍

**Review by Cebion — APPROVED** [review]

**Pull request merged**